src/commands/init.js

```diff
@@ -246,9 +205,7 @@ class InitCommand extends Command {
 
     persistState({ state, siteInfo })
 
-    await (flags.manual ? configManual(this, siteInfo, repo) : configureProvider({ context: this, siteInfo, repo }))
```

Comment:
This was duplicated with https://github.com/netlify/cli/blob/bd59e01b8f4214035e6f69bd85182008bb810f07/src/commands/sites/create.js#L137